app/views.py

```diff
@@ -83,13 +83,38 @@ def logout():
 @app.route('/showAddScript')
 def showAddScript():
 	if session.get('user'):
-		print session['user']
+		#print (session['user'])
 		return render_template('addScript.html',author = session['user'][0])
 	else:
 		return render_template('error.html',error = 'Unauthorized Access')    
 
 
 
+@app.route('/addScript',methods=['POST'])
+def addScript():
+	try:
+		if session.get('user'):
+			_user = session.get('user')[1]
```

Comment:
Could you also give ```_user``` as a parameter to addScript? Then in future we can make a call from Android or Desktop app to save script.